modules/dev-tools/package.json

```diff
@@ -84,7 +84,7 @@
     "lodash.template": "^4.5.0",
     "module-alias": "^2.0.0",
     "nyc": "^15.0.0",
-    "prettier": "2.3.1",
+    "prettier": "2.4.1",
```

Comment:
I had to pin prettier in the OSS repoa due to some issue, maybe 2.4 fixes that?

Reply:
We'll have to pin it in our repo as some of the monorepo workspaces have different versions listed out